README.md

```diff
@@ -82,3 +82,10 @@ complete.
 ### Exempt specific forms from survey continue option
 If the option to allow a survey submission despite incomplete required fields is selected, this
 option allows some forms to be exempted from this.
+
+## Baseline date
+If a date field is selected for this setting, a *recede baseline date* option will be shown next to
+the field when the project is in development status (or if the server is a development server). This
```

Comment:
maybe for next version - allow baseline date receding to apply only to certain records, have a condition e.g. pat_surname='xyz'